packages/runner-ct/cypress/integration/runner.spec.ts

```diff
@@ -0,0 +1,12 @@
+describe('Component testing runner', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:4444/__ct/')
+  })
+
+  it('Opens component testing interface and runs test', () => {
+    cy.wait(500)
+    cy.contains('18n').next().contains('spec.js').click()
```

Comment:
```suggestion
    cy.contains('i18n').next().contains('spec.js').click()
```

Also, what does next() do? (just curious)